scripts/process.py

```diff
@@ -9,10 +9,16 @@
     help="Database connection string",
     default="postgresql://httpd_meta@db3.pcic.uvic.ca/pcic_meta",
 )
-@click.option("-f", "--filename", help="Filename to search for in DB")
-def process(connection_string, filename):
-    datafile = finder.find_filepath(connection_string, filename)
-    print(datafile.filename)
+@click.option("-u", "--unique-id", help="Unique_id to search for in DB")
+@click.option(
+    "-v", "--variable", help="Varible + time range string (ex. tasmax[0:55114])"
+)
+@click.option("-t", "--lat", help="Latitude slab as index (ex. [90:100])")
+@click.option("-n", "--lon", help="Longitude slab as index (ex. [90:100])")
```

Comment:
Would you be able to replace the ex values for these two with the coordinates for Victoria? Could be a fun easter egg.